### PR TITLE
Add dockerfile to deploy storybook

### PR DESCRIPTION
**Pull request by haoict**
https://github.com/davidnguyen179/storybook-wdio/issues/49

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,23 @@
+#########################################################
+# Dockerfile for deploying storybook                    #
+# Usage example:                                        #
+# - Build image:                                        #
+# $ docker build . -t haoict/storybook-wido             #
```
Comment (davidnguyen179) [inline Dockerfile]: What those comments for? Can we remove that?

Comment (haoict) [reply Dockerfile]: Just simple examples how to build and run the images, for noobs

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+COPY . .
+
+# install node app
+RUN npm install
+
+# build storybook
+RUN npm build
+
+# Command
+CMD npm run storybook
```
Comment (davidnguyen179) [inline Dockerfile]: I think this command is for `development`. Here is the production one

```bash
CMD npm run storybook-production
```

File: Dockerfile (hunk 1 above)
Comment (davidnguyen179) [reply Dockerfile]: yeah should remove that!

**Pull request merged**